### ISSUE TITLE
Add empty alt text to logo image

### ISSUE THREAD

**Issue by rshipp** [BanchanArt/banchan#548]
**Issue**: Images must have alternate text
**Impact**: Critical
**Recommendation**: Add `alt=””` to this `img`, so the image is not announced to the screenreader. This is a purely decorative image, and the link will already be announced correctly because the “Banchan Art” text is also inside it.
**More info**: https://dequeuniversity.com/rules/axe/4.6/image-alt 
Screenshot:
![image](https://github.com/BanchanArt/banchan/assets/1382374/43e65aa5-25d6-4d15-babe-a775ec583427)
